app/models/in_progress_disability_compensation_form.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class InProgressDisabilityCompensationForm < InProgressForm
+  EXPIRES_AFTER = 365.days
```

Comment:
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/10128#issuecomment-384118068